std/experimental/allocator/common.d

```diff
@@ -216,6 +220,7 @@ enum uint platformAlignment = std.algorithm.max(double.alignof, real.alignof);
 The default good size allocation is deduced as $(D n) rounded up to the
 allocator's alignment.
 */
+@safe @nogc nothrow pure
```

Comment:
@JakobOvrum I thought I didn't touch any of the function templates, but apparently this one slipped. Fixed & rebased.